runtime/compiler/z/runtime/s390_macros.inc

```diff
@@ -112,8 +112,8 @@ SETVAL(eq_lastSlotField_inInterfaceSnippet,72)
 SETVAL(eq_firstSlot_inInterfaceSnippet,80)
 
 ZZ For single dynamic slot case
-SETVAL(eq_implementorClass_inInterfaceSnippet,48)
-SETVAL(eq_implementorMethod_inInterfaceSnippet,56)
+SETVAL(eq_implementerClass_inInterfaceSnippet,48)
+SETVAL(eq_implementerMethod_inInterfaceSnippet,56)
```

Comment:
This is that one assembly file that's really code as opposed to build, but ...

Reply:
@fjeremic